.github/workflows/python-validate.yml

```diff
@@ -77,3 +77,53 @@ jobs:
 
           python3 -m pip install pytest
           python3 -m pytest
+
+      - name: Install SDK Schema
+        if: steps.pathChanges.outputs.sdkSchema == 'true'
+        run: |
+          cd python/packages/sdk-schema
+
+          python3 -m pip install poethepoet
+          python3 -m poethepoet install
+
+      - name: Validate SDK Schema code formatting
+        run: |
+          cd python/packages/sdk-schema
+
+          python3 -m pip install black
+          python3 -m black --check .
+
+      - name: Check SDK Schema code with linter
+        run: |
+          cd python/packages/sdk-schema
+
+          python3 -m pip install ruff
+          python3 -m ruff .
+
+      - name: Run SDK Schema unit tests
```

Comment:
Shouldn't those tests be run after we have proto changes linted and build?

Reply:
Good point 👍